package.json

```diff
@@ -167,7 +167,7 @@
     "cozy-device-helper": "^1.17.0",
     "cozy-doctypes": "1.82.2",
     "cozy-flags": "2.4.0",
-    "cozy-harvest-lib": "^7.3.1",
+    "cozy-harvest-lib": "8.2.1",
```

Comment:
"cozy-harvest-lib": "^8.2.1" serait mieux, il manque le `^`

Reply:
I don't see a problem. Is it a nit? Why would the `^` help?

Is there a guideline in cozy about that?

ps: @JF-Cozy  should we add this discussion inside a FrontMeeting?

Reply:
oui c'est un nit, j'ai validé la PR en même temps. Oui il y a une guildeline sur ça, les lib cozy sont avec des `^` dans les app cozy.